eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -86,7 +86,7 @@ steps:
   - pwsh: |
       Push-Location ${{ parameters.WorkingDirectory }}/repo
 
-      git checkout *
+      git reset --hard origin/${{ parameters.SourceBranchName }}
```

Comment:
Assuming the goal is to just undo any changes no matter what branch you are on you can simply use `HEAD` as the branch, but if your goal is to reset the current branch to SourceBranchName then what you have will do the trick.

Reply:
In this case, both will do the same thing.  We've checked out `<sourcebranchname>`, made changes to it, then pushed those changes to a new upstream. 

This means that `origin/<sourcebranchname>` _is_ directly equivalent to HEAD. 

All that being said, I haven't re-run my checks, so I'll update to `HEAD`.